index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const greeting = require('./lib/greet.js');
+
+greeting.greet(process.argv[2]);
```

Comment:
You could have also stored process.argv[2] in a variable and pass it as an argument to a separate function, see below for further elaboration on this!